cartography/data/jobs/cleanup/gcp_sql_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+    "statements": [
+      {
+        "query": "MATCH (i:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{accountid: {GCP_PROJECT_ID}}) WHERE i.lastupdated <> {UPDATE_TAG} WITH i LIMIT {LIMIT_SIZE} DETACH DELETE (i) return COUNT(*) as TotalCompleted",
+        "iterative": true,
+        "iterationsize": 100,
+        "__comment__": "Delete GCP Cloud SQL Instance that no longer exist and detach them from all previously connected nodes."
+      },
+      {
+        "query": "MATCH (:GCPSQLInstance)<-[r:RESOURCE]-(:GCPProject{accountid: {GCP_PROJECT_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+        "iterative": true,
+        "iterationsize": 100,
+        "__comment__": "Remove GCP GCPSQLInstance-to-Project relationships that are out of date."
+      },
+      {
+        "query": "MATCH (u:GCPSQLUser)<-[:RESOURCE]-(i:GCPSQLInstance{id:instance.id}) WHERE u.lastupdated <> {UPDATE_TAG} WITH u LIMIT {LIMIT_SIZE} DELETE (u) return COUNT(*) as TotalCompleted",
```

Comment:
We should add relationship all the way through GCPProject. This should be something like:
`(u:GCPSQLUser)<-[:RESOURCE]-(:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{accountid: {GCP_PROJECT_ID}})`